components/match2/commons/opponent.lua

```diff
@@ -237,16 +237,25 @@ Resolves the identifiers of an opponent.
 For team opponents, this resolves the team template to a particular date. For
 party opponents, this fills in players' pageNames using their displayNames,
 using data stored in page variables if present.
+
+options.loadFlag: Whether to fetch the flag from variables, match2 or LPDB. Disabled by default
```

Comment:
this is not correct

`PlayerExt.syncPlayer(player, options)` only tries to fetch from match2player data samples if explicitly enabled by `options.fetchMatch2Player`

(this fetching from match2player is btw not what would happen with the automation i plan, there it would just use the data from the match/standing that defines the placement, no sync needed at all)